tests/main/proxy-no-core/task.yaml

```diff
@@ -0,0 +1,42 @@
+summary: Ensure that the core.proxy.* settings are honored without core
+
+# only needs a test on classic
+systems: [ubuntu-16.04-64, ubuntu-18.04-64]
+
+restore: |
+    iptables -D OUTPUT -m owner --uid-owner "$(id -u test)" -j ACCEPT -p tcp
+    iptables -D OUTPUT -j REJECT -p tcp --dport http --reject-with tcp-reset
+    iptables -D OUTPUT -j REJECT -p tcp --dport https --reject-with tcp-reset
+
+execute: |
+    if ! command -v python3; then
+       echo "SKIP: need python3"
+       exit 0
+    fi
+    #shellcheck source=tests/lib/systemd.sh
+    . "$TESTSLIB"/systemd.sh
+    systemctl stop snapd.service snapd.socket
+    rm -rf /var/lib/snapd/state.json
+    systemctl start snapd.service snapd.socket
+
+    # only allow test user to connect to http/https
+    iptables -I OUTPUT -j REJECT -p tcp --dport http --reject-with tcp-reset
+    iptables -I OUTPUT -j REJECT -p tcp --dport https --reject-with tcp-reset
+    iptables -I OUTPUT -m owner --uid-owner "$(id -u test)" -j ACCEPT -p tcp
+    systemd-run --uid=test --unit tinyproxy -- python3 "$TESTSLIB/tinyproxy/tinyproxy.py"
```

Comment:
What ensures that this is stopped?

Reply:
Thanks, this was indeed missing, fixed now.